src/component/handlers/edit/editOnKeyDown.js

```diff
@@ -101,6 +107,19 @@ function editOnKeyDown(e: SyntheticKeyboardEvent): void {
     case Keys.DOWN:
       this.props.onDownArrow && this.props.onDownArrow(e);
       return;
+    case Keys.SPACE:
+      // handling for OSX where option + space scrolls
+      if (isChrome && isOptionKeyCommand(e)) {
+        e.preventDefault();
+        // insert a space into the editor
+        const contentState = DraftModifier.replaceText(
+            editorState.getCurrentContent(),
+            editorState.getSelection(),
+            ' '
+        );
+        this.update(EditorState.push(editorState, contentState, 'space'));
```

Comment:
The correct `EditorChangeType` here is `'insert-characters'`. (I need to document this, but one should always stick to the `EditorChangeType` values for `push` calls.)